modules/tensor_mechanics/include/materials/RadialReturnStressUpdate.h

```diff
@@ -70,13 +70,13 @@ class RadialReturnStressUpdateTempl : public StressUpdateBaseTempl<is_ad>,
                            bool /*compute_full_tangent_operator*/,
                            RankFourTensor & /*tangent_operator*/) override;
```

Comment:
here's crazy idea: provide default values for the last two parameters and have no overloads.
The default for the tangent operator reference could be a static member

Reply:
I wonder if the last to arguments could just be folded into one by making `tangent_operator` a pointer (that defaults to `nullptr` in the case where `compute_full_tangent_operator` would be false.

Reply:
`MultiParameterPlasticityStressUpdate::updateState` uses those two arguments independently.